### PR TITLE
PATCH /orders/:order_id/address to update an order's address

### PR DESCRIPTION
**Pull request by CharlotteW923**
## Description

Allows admins to modify the address of an order by providing the order ID and updated address details.

## Changes

- Adds the PATCH /orders/:order_id/address endpoint
- Gives only admins access to the endpoint
- Updates the order_updates table after an order's address is successfully updated

## Screenshots/demo

Before update:
image
After Update:
image
image
order_updates table:
image

## Related issues

What issue(s) does this PR fix?
#42

### CODE REVIEW

File: api/types/payloads.go
Comment (sebastian-nunez) [inline api/types/payloads.go:58]: nit: make it "aptUnit"

File: api/services/orders/orders_store.go
Comment (sebastian-nunez) [inline api/services/orders/orders_store.go:17]: nit: I don't see it used anywhere right now. Are you planning to check that the order exists before updating?

File: api/types/payloads.go
Comment (sebastian-nunez) [inline api/types/payloads.go:59]: Good enum!

File: api/types/stores.go
Comment (sebastian-nunez) [inline api/types/stores.go:25]: nit: reformat looks like a space is missing

**Review by sebastian-nunez — APPROVED** [review]
Great work! LGTM after the minor changes

**Comment by sebastian-nunez** [comment]
@CharlotteW923 hold off on merging until I have Abel's PR wrapped up

File: api/types/payloads.go
Comment (CharlotteW923) [reply api/types/payloads.go:58]: I get this error:
image
I'm reading that it needs to start with uppercase for it to work

File: api/types/stores.go
Comment (CharlotteW923) [reply api/types/stores.go:25]: Fixed I think? I don't see any weird spacing on my end
image

Comment (sebastian-nunez) [reply api/types/stores.go:24]: weird. maybe it was because I was on my phone then idk. Looks good tho

File: api/services/orders/orders_store.go
Comment (sebastian-nunez) [reply api/services/orders/orders_store.go]: sounds good

File: api/types/payloads.go
Comment (sebastian-nunez) [reply api/types/payloads.go:58]: sorry the `json` part like this `json:"aptUnit"` just to be consistent with the rest of the code

Comment (CharlotteW923) [reply api/types/payloads.go:58]: Fixed! Should we not be writing it how its listed in the database schema file?

Comment (sebastian-nunez) [reply api/types/payloads.go:58]: Typically, JSON you always use the camelCase since that's the standard notation used JavaScript/TypeScript

**Pull request merged**
